cmd/api/http/http_test.go

```diff
@@ -18,13 +21,28 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+func TestMain(m *testing.M) {
+	var err error
+
+	reqTimeoutStr := os.Getenv("HTTP_REQUEST_TIMEOUT") //This ENV must be written with a unit suffix, like seconds
+	if reqTimeoutStr != "" {
+		book.RequestTimeout, err = time.ParseDuration(reqTimeoutStr)
+		if err != nil {
+			log.Fatalln("getting request timeout from env: %w", err)
+		}
+	}
```

Comment:
Same thing about the timeout not being a global and actually being part of the `BookHandler`.

Reply:
Ok, done. But if we want change the timeout value to watch the behavior of tests, wouldn't it be nice to get that value trought environment?

Reply:
Your tests should receive the context for any given method call, you can setup the context to be a `WithTimeout` with the timeout you would want.